depenguinme.sh

```diff
@@ -95,13 +95,8 @@ REQUIRE_SSHKEY=YES
 DAEMONIZE=NO
 USE_IPV6=NO
 MFSBSDISO="https://depenguin.me/files/mfsbsd-14.0-RELEASE-amd64.iso"
-DEPS=(
-  "mkisofs:mkisofs"
-  "qemu-system-x86_64:qemu-system-x86"
-  "kvm-ok:cpu-checker"
-  "ovmf:ovmf"
-  )  # binary:package
 
+# usage
```

Comment:
Is this comment adding much value? What about "# display command usage" or something like that?